android9/overlay/etc/sensorfw/sensord.conf.d/sensord.conf

```diff
@@ -0,0 +1 @@
+# To be overridden by Halium overrides
```

Comment:
Does it make sense to provide a default instantiation of the hybris plugins here?

Reply:
https://github.com/erfanoabdi/rootfs-builder-debos-android9/commit/9ab939a463eddbd0857027766c3ffa268d5711f2

Reply:
I would prefer @erfanoabdi's way for now. Marius tried to handle sensorfw configuration via https://github.com/ubports/deviceinfo instead, so it's probably better to keep changes in one place until we figure out how to use that.